paasta_tools/setup_kubernetes_services.py

```diff
@@ -0,0 +1,183 @@
+#!/usr/bin/env python
+# Copyright 2015-2018 Yelp Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Usage: ./setup_kubernetes_services.py <service.instance>
+
+Command line options:
+
+- -c, --cluster: PAASTA cluster
+- -v, --verbose: Verbose output
+"""
+import argparse
+import logging
+import sys
+from typing import Sequence
+
+from kubernetes.client import V1APIService
+from kubernetes.client import V1ObjectMeta
+from kubernetes.client import V1Service
+from kubernetes.client import V1ServicePort
+from kubernetes.client import V1ServiceSpec
```

Comment:
it's going to become cumbersome to import every single entity separately, let's just `import kubernetes.client as k8s`?